### ISSUE TITLE
Stop forcing Infima, an unknown, unpopular CSS framework with less than 500 stars, on Docusaurus users as the default! Provide 3-4 popular default options of using the Most common & popular CSS Frameworks (e.g. Tailwind, Bulma, Bootstrap5)!

### ISSUE THREAD

**Issue by pmeaney** [facebook/docusaurus#9738]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Motivation

I am running into the problem with Docusaurus of:

1. Docs & Blog pages are not in /src/pages directory.  Yet, to create a new custom page, Docusaurus places them there.  OK Docusaurus Devs, why not follow your own convention and place the Docs & Blogs pages there as well, so I can edit them?

2. Why do I want to edit them? So I can remove everything InfimaCSS (https://github.com/facebookincubator/infima) from the project.  Why would I use Infima?  In fact, why would the Docusaurus team choose Infima-- a CSS framework which barely anyone uses or knows?  

It's a contrived attempt to get a bit of market share towards InfimaCSS-- a Meta-created CSS Framework.  Perhaps provide it as an option-- for example, when a NextJS project is created via CLI, we are prompted with "Use TailwindCSS? Y/N".  

Docusaurus could similarly ask: "Pick your default CSS Framework.  Tailwind / Bulma / Bootstrap / Infima".  

# Github Stars (as of Jan 15, 2024)
## Bootstrap5: 166,000+
# ....Infima: 387 <-- you stuck us with that?!

**WHAT?!** Docusaurus designers/devs have chosen to stick us with an unknown, unpopular CSS Framework, by default?!  Seriously?

**And to make it even worse, they've made it difficult to access Docs & Blogs pages?!**  

Wow, those are two horrible system design choices:
1. src/page does not contain Blogs.js nor Docs.js
2. Since I can't access those two pages, I can't change their divs' classes! Making it difficult to use a non-Infima CSS Framework across my entire Docusaurus site.

### Self-service

- [ ] I'd be willing to do some initial work on this proposal myself.

**Comment by Josh-Cena** [comment]
> In fact, why would the Docusaurus team choose Infima

Because Infima is just part of Docusaurus, extracted to a separate package... Docusaurus created Infima to be part of Docusaurus in the first place. We could have baked Infima as part of the classic theme, we just decided to make a separate project.

**Comment by pmeaney** [comment]
Thanks @Josh-Cena Makes sense! I see Infima is part of a Meta incubator: https://github.com/facebookincubator/infima
That's fantastic to support a project in its infancy.  

However, I would greatly prefer to have easy access to other options-- particularly: Popular options, **so that I can continue building knowledge & skills around CSS frameworks which are actually well used & well liked in the Frontend Developer marketplace.**

**Comment by Josh-Cena** [comment]
> That's fantastic to support a project in its infancy

It makes no sense to me. We are not "supporting" Infima. We ARE Infima. Just picture another world where Infima doesn't exist and all the CSS stylesheets are directly contained in the classic theme. It's the exact same outcome for you as a user. We just decided to put the stylesheets into a separate package.

We will be making a Tailwind theme (https://github.com/facebook/docusaurus/issues/2961). We have made a Bootstrap theme but it was neither popular nor maintained. If you want to see a solution, consider either making one yourself or sponsoring someone to make it happen.
